src/log-manager/src/cleaner/entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+set -o errexit
+set -o pipefail
+
+log_exist_time=30 # 30 day
```

Comment:
So a job's log will be removed after 30 days? Currently we don't clean user's job in database. If a user views his previous job after it completes for 30 days, and the job log is cleaned, what will happen?

Reply:
They can not retrieve the job log. 404 will return. If they want to persistent job log, we recommend customer use external storage, such as AzureBlob or NFS (It's next step, we don't implement yet).

Reply:
If applicable, I think it's better to remind users that their logs will be removed after 30 days in the webportal. It's strange that a previous existing log just disappears and report a 404 error.

Reply:
It's the same behavior with previous version. We could add doc for this. For webportal change, it's another enhance and we could open another issue to track

Reply:
Agreed. Please open an issue for it.

Resolved this conversation.